### PR TITLE
Add workflow to create release

### LINKED ISSUE #64: Implement github action to run a geno release
As new content is added to geno-edit.owl to address community requests, we need a way to ensure that it propagates to the version of the ontology visible / useable to the community (e.g. in services/resources like OLs, Ontobee).  My understanding is that a release needs to be cut in order for these services to pick up new content.  

@matentzn suggested implementing a github action to allow for the release process to be initiated in a more automated way.

### PR DESCRIPTION
**Pull request by anitacaron**
Fixes #64

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  push:
+    branches: [ develop ]
+    paths:
+      - 'src/ontology/imports/**_import.owl'
+      - 'src/ontology/components/**.owl'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: obolibrary/odkfull:v1.4
```
Comment (matentzn) [inline .github/workflows/build.yml]:
```suggestion
    container: obolibrary/odkfull:v1.4.3
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: Build release files
+on:
+  workflow_dispatch:
+  push:
+    branches: [ develop ]
+    paths:
+      - 'src/ontology/imports/**_import.owl'
+      - 'src/ontology/components/**.owl'
```
Comment (matentzn) [inline .github/workflows/build.yml]: What about the edit file?

Comment (anitacaron) [reply .github/workflows/build.yml]: Is your intention to have a release after each PR is merged? How about when two PRs are merged on the same day?

Comment (matentzn) [reply .github/workflows/build.yml]: you are right. maybe lets just remove the branch trigger altogether and instead only support `workflow_dispatch`

**Review by matentzn — APPROVED** [review]

**Pull request merged**